services/core-api/app/api/mines/permits/permit_conditions/models/standard_permit_conditions.py

```diff
@@ -27,11 +26,47 @@ class StandardPermitConditions(SoftDeleteMixin, AuditMixin, Base):
     parent_standard_permit_condition_id = db.Column(
         db.Integer, db.ForeignKey('standard_permit_conditions.standard_permit_condition_id'))
     display_order = db.Column(db.Integer, nullable=False)
-    sub_conditions = db.relationship(
+    all_sub_conditions = db.relationship(
         'StandardPermitConditions',
         lazy='joined',
+        order_by='asc(StandardPermitConditions.display_order)',
         backref=backref('parent', remote_side=[standard_permit_condition_id]))
 
+    def __repr__(self):
+        return '<StandardPermitConditions %r, %r>' % (self.standard_permit_condition_id,
```

Comment:
There is a way to get the class name automatically. Can you adjust it to do that? Look at other "repr" functions in other classes for an example.